micro-docker/README.md

```diff
@@ -0,0 +1,4 @@
+- cd micro-docker
+- sudo docker build -t rackhd/micro .
+- sudo docker save rackhd/micro | xz -z > micro.tar.xz
+- copy micro.tar.xz to on-http/static/http
```

Comment:
This still needs to be changed to match the on-tasks name, right? https://github.com/RackHD/on-tasks/pull/293#issuecomment-236190792